Mathlib/LinearAlgebra/AffineSpace/Independent.lean

```diff
@@ -841,6 +841,25 @@ theorem range_face_points {n : ℕ} (s : Simplex k P n) {fs : Finset (Fin (n + 1
     (h : #fs = m + 1) : Set.range (s.face h).points = s.points '' ↑fs := by
   rw [face_points', Set.range_comp, Finset.range_orderEmbOfFin]
 
+/-- The face of a simplex with all but one point. -/
+def faceOpposite {n : ℕ} [NeZero n] (s : Simplex k P n) (i : Fin (n + 1)) : Simplex k P (n - 1) :=
+  s.face (fs := {j | j ≠ i}) (by simp [filter_ne', NeZero.one_le])
+
+@[simp] lemma range_faceOpposite_points {n : ℕ} [NeZero n] (s : Simplex k P n) (i : Fin (n + 1)) :
+    Set.range (s.faceOpposite i).points = s.points '' {j | j ≠ i} := by
+  simp [faceOpposite]
+
+lemma mem_affineSpan_range_face_points_iff [Nontrivial k] {n : ℕ} (s : Simplex k P n)
```

Comment:
This could also be a simp-lemma, right?
```suggestion
@[simp] lemma mem_affineSpan_range_face_points_iff [Nontrivial k] {n : ℕ} (s : Simplex k P n)
```

Reply:
Because `range_face_points` is a `simp` lemma, this one isn't in `simp` normal form (and likewise for `mem_affineSpan_range_faceOpposite_points_iff`).